warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -707,7 +708,7 @@ protected ASTJexlScript updateQueryTree(ScannerFactory scannerFactory, MetadataH
         stopwatch.stop();
         
         // Find unmarked bounded ranges
-        if (UnmarkedBoundedRangeDetectionVisitor.findUnmarkedBoundedRanges(queryTree)) {
+        if (UnmarkedBoundedRangeDetectionVisitor.findUnmarkedBoundedRanges(config.getQueryTree())) {
             throw new DatawaveFatalQueryException("Found incorrectly marked bounded ranges");
         }
         
```

Comment:
need a config.getQueyrTree on line 718 instead of old reference to queryTree, right?

Reply:
Good catch, thanks.  The github "resolve conflicts" brought these new lines in.  I just updated them and don't see any other additions that need updating.